test/blocks/editorial-card/editorial-card.test.js

```diff
@@ -34,7 +34,7 @@ describe('editorial-card', () => {
   });
 
   it('w/ 4 rows has a footer', () => {
-    const footer = editorialCards[3].querySelector('.footer');
+    const footer = editorialCards[3].querySelector('.card-footer');
     expect(footer).to.exist;
```

Comment:
Nit: This could be a 1 liner.
```suggestion
    expect(editorialCards[3].querySelector('.card-footer')).to.exist;
```